### PR TITLE
Fix CI after changing master branch name to main

### PR DESCRIPTION
**Pull request by kjvalencik**

### CODE REVIEW

File: crates/neon-sys/native/src/neon_class_metadata.h [hunk 1]
```diff
@@ -20,7 +20,7 @@
 // Slots 0 and 1 appear to be reserved by Chrome, and slot 3 is reserved by Node.
 // That apparently leaves only slot 2 available for storing Neon metadata.
 //
-// See: https://github.com/nodejs/node/blob/master/src/env.h#L33-L39
+// See: https://github.com/nodejs/node/blob/main/src/env.h#L33-L39
```
Comment (goto-bus-stop) [inline crates/neon-sys/native/src/neon_class_metadata.h:23]: this one shouldnt be changed :D

```suggestion
// See: https://github.com/nodejs/node/blob/master/src/env.h#L33-L39
```

**Review by goto-bus-stop — APPROVED** [review]
oh boy, i grepped through everything so many times to make sure, but of course `rg` ignores dotfiles by default… thanks!

File: crates/neon-sys/native/src/neon_class_metadata.h (hunk 1 above)
Comment (kjvalencik) [reply crates/neon-sys/native/src/neon_class_metadata.h:23]: 🤦

**Review by dherman — COMMENTED** [review]
:shipit:

**Review by dherman — COMMENTED** [review]
:shipit:

**Comment by dherman** [comment]
oh lol I was wondering why I couldn't review it! it's already merged 😂

**Pull request merged**